operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,404 @@
+# Editing/Deleting critical services from static ConfigMap (`rrs-mon-static`)
+
+## Overview
```

Comment:
This overview should also provide an overview of what this page contains. (procedures to list, delete, and modify the critical services list).